ocsci/main.py

```diff
@@ -106,11 +107,15 @@ def render_yaml_with_j2_context(yaml_data):
     return yaml.safe_load(out)
 
 
-def main(arguments):
+def main():
+    arguments = sys.argv[1:]
     init_ocsci_conf(arguments)
     arguments.extend([
         '-p', 'ocsci.pytest_customization.ocscilib',
         '-p', 'ocsci.pytest_customization.marks',
         '-p', 'ocsci.pytest_customization.ocsci_logging',
     ])
-    return pytest.main(arguments)
+    from ocsci import config
+    from utility.utils import add_path_to_env_path
+    add_path_to_env_path(os.path.expanduser(config.RUN['bin_dir']))
+    exit(pytest.main(arguments))
```

Comment:
return ?

Reply:
After discussion with Martin, I've decided not to remove the `run_ocsci.py` wrapper in this PR.